assets/src/styles/blocks/Articles.scss

```diff
@@ -57,6 +57,10 @@
 .article-list-item-image {
   order: 2;
 
+  a {
```

Comment:
Before I removed `d-flex`, the focus state of the image link was entirely invisible.

After removing, I had to add this rule. Otherwise the link was actually wider than the image it contains, causing a weird "protrusion" when the link is focused.
![Screenshot from 2021-11-04 15-41-21](https://user-images.githubusercontent.com/7604138/140334460-82b62021-dd08-4ba4-a8d6-381581710eb8.png)

Reply:
That being said, this focused state doesn't play very well around image links. In many cases the contrast between the border and the edge of the image is not sufficient or almost 0.

It still seems better than no focus indication at all, so maybe we can proceed with these changes and improve on it later.